lib/shopify-cli/version.rb

```diff
@@ -1,3 +1,4 @@
 module ShopifyCli
+  # If you are updating this value, consider updating it in packaging/debian as well
```

Comment:
This sort of comment belongs in the Contributing docs for how to release a new version.